pkg/pod/pod.go

```diff
@@ -446,8 +446,9 @@ func (b *Builder) Build(ctx context.Context, taskRun *v1.TaskRun, taskSpec v1.Ta
 		if err != nil {
 			return nil, err
 		}
-		svMinorInt, _ := strconv.Atoi(sv.Minor)
+		svMinor := strings.TrimSuffix(sv.Minor, "+") // Remove '+' if present
 		svMajorInt, _ := strconv.Atoi(sv.Major)
+		svMinorInt, _ := strconv.Atoi(svMinor)
```

Comment:
This looks good, thank you.
It would be nice to have tests for this to avoid regressions in future, but today we don't have a test with a k8s version that includes a plus. 

Perhaps this code block could be a function in `pod.go` so that it can be used in `taskrun.go` as well and have a simple unit test, wdyt?

Reply:
Yes, makes sense @afrittoli, will work on it, thank you.

Reply:
Change pushed, can you check it please @afrittoli

Reply:
Great, thank you!